CMakeLists.txt

```diff
@@ -4,6 +4,7 @@ project(Cucumber-Cpp)
 
 set(CUKE_USE_STATIC_BOOST ${WIN32} CACHE BOOL "Statically link Boost (except boost::test)")
 set(CUKE_USE_STATIC_GTEST ON CACHE BOOL "Statically link Google Test")
+set(CUKE_ENABLE_SHARED_LIB OFF CACHE BOOL "Generate a shared library")
```

Comment:
I've had a look at the branch in build-system. If you take a look at the `CukeExport.hpp`, you'll see that we only need one, not one for static and one for dynamic linking. In static builds, it's sufficient to set `CUCUMBER_CPP_STATIC_DEFINE` from the `gypi`.